docs/clientdev/overview.rst

```diff
@@ -191,26 +191,26 @@ header, it's possible to send these values via the querystring::
 
 .. describe:: sentry_client
 
-    An arbitrary string which identifies your client, including its version.
+    An arbitrary string which identifies your SDK, including its version.
 
     The typical pattern for this is '**client_name**/**client_version**'.
 
-    For example, the Python client might send this as 'raven-python/1.0'.
+    For example, the Python SDK might send this as 'raven-python/1.0'.
 
 .. describe:: sentry_timestamp
 
     The unix timestamp representing the time at which this event was generated.
 
 .. describe:: sentry_key
 
-    The public key which should be provided as part of the client configuration.
+    The public key which should be provided as part of the SDK configuration.
 
 .. describe:: sentry_secret
 
-    The secret key which should be provided as part of the client configuration.
+    The secret key which should be provided as part of the SDK configuration.
 
     .. note:: You should only pass the secret key if you're communicating via
-              secure communication to the server. Client-side behavior (such
+              secure communication to the server. SDK-side behavior (such
```

Comment:
probably correct to leave it as "Client-side" here, since we're using client in the client/server sense